.travis.yml

```diff
@@ -1,22 +1,41 @@
-language: python
-python:
-  - "3.6"
-  - "3.7"
-  - "3.8"
-addons:
-  apt:
-    update: true
-env:
-  - PACKAGE_NAME=pytest-confluence-report PACKAGE_VERSION=0.0.2
-before_install:
-  - sudo add-apt-repository -y ppa:duggan/bats
-  - sudo apt-get -y install bats
-install:
-  - pip install pip -U && pip install -r requirements-dev.txt -U
-script:
-  - bats test-package.bats
-  - ./analyse-source-code.sh
-after_success:
-  - coveralls
-notifications:
-  email: false
+build:
```

Comment:
Regarding commit message. Please make it more readable and meaningful as `try to ci` does not describe the task.

Also, I see 3 separate commits, please squash them into a single one.